### ISSUE TITLE
Test Summary Duration is wrong as it seems to be calculated in seconds instead of milliseconds

### ISSUE THREAD

**Issue by AlbertoBartolomeET** [ctrf-io/github-actions-ctrf#8]
It seems (at least in my cases) that the Duration in Summary is wrong calculated, as it is calculated in seconds instead of milliseconds. Right now it is saying that the only test I have run takes around 10 seconds, but the whole duration is 3 hours XD

Summary: 
- start: 1722511783500
- stop: 1722511794528
- difference: 11028
   - in ms it means 11 seconds and 28 ms
   - in seconds it means 183.8 minutes.... that is 3:03:48 

Test:
- start: 1722511783
- stop: 1722511794
- duration: 10276
- difference (in seconds): 11 seconds, pretty close to duration.

From the json file: 

```
    "summary": {
      "tests": 1,
      "passed": 1,
      "failed": 0,
      "pending": 0,
      "skipped": 0,
      "other": 0,
      "start": 1722511783500,
      "stop": 1722511794528,
      "suites": 0
    },
    "tests": [
      {
        "name": "Login - Logout @smoke @auth",
        "status": "passed",
        "duration": 10276,
        "start": 1722511783,
        "stop": 1722511794,
        "rawStatus": "passed",
        "tags": [
          "@smoke",
          "@auth"
        ],
```

<img width="773" alt="image" src="https://github.com/user-attachments/assets/e0245b02-1ac9-4071-967f-d1611657edf4">

**Comment by Ma11hewThomas** [comment]
That's embarrassing! Thanks for reporting! Released fix in 0.0.21

**Comment by AlbertoBartolomeET** [comment]
@Ma11hewThomas not at all, many thanks for all the work and the super fast fix!!
